src/components/NewsList/NewsItem/style.css

```diff
@@ -0,0 +1,15 @@
+.NewsItem {
+  padding: 20px;
+  margin: 12px;
+  cursor: pointer;
+  list-style: none;
+  text-align: left;
+
+  border-radius: 12px;
+  background: linear-gradient(145deg, #d4d4d4, #fbfbfb);
+  box-shadow: 5px 5px 10px #5e5e5e, -5px -5px 10px #ffffff;
```

Comment:
Create global variables for colors.

Reply:
folder name should always be cameCase i.e fooBar.

Only component file name are allowed uppercase i.e FooBar.js